include/exadg/convection_diffusion/preconditioners/multigrid_preconditioner.cpp

```diff
@@ -281,11 +281,14 @@ template<int dim, typename Number>
 void
 MultigridPreconditioner<dim, Number>::set_velocity(VectorTypeMG const & velocity)
 {
+  unsigned int const fine_level   = this->get_number_of_levels() - 1;
+  unsigned int const coarse_level = 0;
```

Comment:
I guess `coarse_level = 0` is per convention in ExaDG? In deal.II we can terminate at any level.

Reply:
Yes, it was hard-coded to 0 before. So I suggest to introduce this again once that functionality is really implemented.